README.md

```diff
@@ -155,10 +147,10 @@ For instructions on using this tutorial, see [Instructions](https://github.com/d
    2. .NET Core cross-platform development
    3. Visual Studio extension development
 2. Clone this repository
-3. Install .NET Core SDK version specified in `.\global.json` with `"dotnet":` from [here](https://dotnet.microsoft.com/download/dotnet-core).
+3. Install .NET SDK version specified in `.\global.json` with `"dotnet":` from [here](https://dotnet.microsoft.com/download/dotnet-core).
 4. Open a command prompt and go to the directory of the Roslyn Analyzer Repo
 5. Run the restore and build command: `build.cmd`(in the command prompt) or `.\build.cmd`(in PowerShell).
```

Comment:
Might be worth mentioning `Restore.cmd`.

Reply:
@Evangelink `Build.cmd` does both restore and build:

https://github.com/dotnet/roslyn-analyzers/blob/f471d3381584f10f9908432e0b2b2b8ef07a0aa6/Build.cmd#L2